src/components/application_manager/test/commands/mobile/reset_global_properties_test.cc

```diff
@@ -308,7 +337,7 @@ TEST_F(ResetGlobalPropertiesRequestTest,
   (*msg_)[am::strings::msg_params][am::strings::properties][0] =
       mobile_apis::GlobalProperty::TIMEOUTPROMPT;
   (*msg_)[am::strings::msg_params][am::strings::properties][1] =
-      mobile_apis::GlobalProperty::MENUICON;
+      mobile_apis::GlobalProperty::MENUNAME;
```

Comment:
@dcherniev in couple of files you changes all menu_name to menu icon, there  you do conversely. Is it error?